### ISSUE TITLE
placeOrder not working with v9.71 API 

### ISSUE THREAD

**Issue by okonovalenko** [pilwon/node-ib#4]
Hi, great library first of all, I will be the first one to raise an issue.
When I call placeOrder there is no response from the server at all in the logs, doesn't even reply with correct format error.

Regards

**Comment by junajan** [comment]
Hello, I'm using it with TWS Build 949.3e, May 12, 2015 10:56:46 AM and it works fine.. limit orders will pop up in tws log correctly..

**Comment by andrewjdmartin** [comment]
@okonovalenko I realize that this is quite old at this point, but I was running into the same issue. I wasn't using the .order builder, but rather just creating an object manually, as I wanted to use order types other than limit. My mistake was not specifying **transmit: true** in the order object. This results in the order being stored locally but never transmitted to IB.

e.g.:

````
var orderObj = {
    action: 'BUY',
    auxPrice: 0,
    lmtPrice: 0,
    orderType: 'MKT',
    totalQuantity: 1,
    transmit: true
};
ib.placeOrder( nextValidId, ib.contract.stock( 'AAPL', 'SMART', 'USD' ), orderObj );
````

**Comment by okonovalenko** [comment]
Thanks for that 😊 

-----Original Message-----

**Comment by pilwon** [comment]
Thanks, the related PR (#16) is now merged and published as v0.1.5.
